dynacred/src/Contact.ts

```diff
@@ -80,7 +94,7 @@ export class Contact {
     }
 
     set email(e: string) {
-        if (e) {
+        if (this.email !== e) {
```

Comment:
You're doing the check for empty value in `alias` but not in `email`, `phone` or `url`, wanted?

Reply:
yes, alias needs to be non-empty, email, phone, and url can be empty.